mkver.conf

```diff
@@ -0,0 +1,80 @@
+tagPrefix: v
```

Comment:
i'm not very familiar with mkver. can you write a short summary explaining this?

Reply:
Basically mkver just doing semantic versioning for us, its using conventional commits for it. This config specifies settings for semantic gersioning like tag format, version format, files to automatically patch.
In short words, same thing as bumpversion. Actually we might use bumpversion instead of mkver, but need to tune it, so helm charts will have it's own versioning